app/views/RoomInfoView/index.js

```diff
@@ -94,6 +97,25 @@ export default class RoomInfoView extends React.Component {
 			const roomUserId = RocketChat.getRoomMemberId(this.rid, user.id);
 			try {
 				const result = await RocketChat.getUserInfo(roomUserId);
+				/*
+					isFollowing is true if the user is following the fetched user, false otherwise.
+					isFollowing is added as a property under the user which itself is a property of the result.
+				*/
+
+				const followersOfTheUser = await RocketChat.getFollowers(result.user.username);
+				const followingOfTheUser = await RocketChat.getFollowing(result.user.username);
+				if (followersOfTheUser) {
```

Comment:
This can also be re written as: result.user.isFollowing = (followersOfTheUser[user.id] === '') ? true : false